### PR TITLE
Adding resources for the ECS cross-account blog post

### PR DESCRIPTION
**Pull request by natronq**
*Issue #, if available:*

*Description of changes:*
Adding resources for the upcoming ECS cross-account blog post.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: blogs/ecs-cross-account/account_backend/redis_and_database.yml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+      SecurityGroupIngress:
+        - CidrIp:
+            Fn::ImportValue: !Sub "${ProjectName}:VpcCIDR"
+          IpProtocol: -1
+
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "${ProjectName}-log-group"
+      RetentionInDays: 30
```
Comment (jlbutler) [inline blogs/ecs-cross-account/account_backend/redis_and_database.yml:43]: This is for a demo, make retention days less as the costs can add up. No more than a couple of days IMO

File: blogs/ecs-cross-account/account_backend/redis_and_database.yml [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+      DnsConfig:
+        NamespaceId:
+          Fn::ImportValue: !Sub "${ProjectName}:ECSServiceDiscoveryNamespace"
+        DnsRecords:
+          - Type: A
+            TTL: 300
+      HealthCheckCustomConfig:
+        FailureThreshold: 1
+
+  DatabaseTaskDef:
```
Comment (jlbutler) [inline blogs/ecs-cross-account/account_backend/redis_and_database.yml:102]: I think all of the TaskDef's need a dependsOn for the envoy container, so the service doesn't come up ahead of the proxy. e.g.

```
"dependsOn": [
        {
          "containerName": "envoy",
          "condition": "HEALTHY"
        }
      ]
```

**Review by jlbutler — CHANGES_REQUESTED** [review]
I haven't run through this but looks fine. Left a few comments to address.

File: blogs/ecs-cross-account/account_backend/redis_and_database.yml [hunk 3]
```diff
@@ -0,0 +1,331 @@
+Parameters:
+  ProjectName:
+    Type: String
+    Description: Project name to link stacks
+    Default: am-ecs-multi-account
+
+  EnvoyImage:
+    Type: String
+    Description: Envoy container image
+    Default: 840364872350.dkr.ecr.eu-central-1.amazonaws.com/aws-appmesh-envoy:v1.12.5.0-prod
```
[the remark this answers is not in the document]
Comment (natronq) [reply blogs/ecs-cross-account/account_backend/redis_and_database.yml]: Good point, changed the code-snippet in the post a bit and made it dynamic.

File: blogs/ecs-cross-account/account_backend/redis_and_database.yml (hunk 1 above)
Comment (natronq) [reply blogs/ecs-cross-account/account_backend/redis_and_database.yml]: Done, 5 days now.

File: blogs/ecs-cross-account/account_backend/redis_and_database.yml (hunk 2 above)
Comment (natronq) [reply blogs/ecs-cross-account/account_backend/redis_and_database.yml:101]: Added dependsOn for x-ray and envoy to every taskdef.

**Review by jlbutler — APPROVED** [review]

**Comment by jlbutler** [comment]
LGTM, thanks for changes and additional testing!

**Pull request merged**